drivers/unix/ip_unix.cpp

```diff
@@ -101,7 +101,7 @@ IP_Address IP_Unix::_resolve_hostname(const String &p_hostname, Type p_type) {
 		hints.ai_family = AF_UNSPEC;
 		hints.ai_flags = AI_ADDRCONFIG;
 	};
-	hints.ai_flags &= !AI_NUMERICHOST;
+	hints.ai_flags &= ~AI_NUMERICHOST;
```

Comment:
:alarm_clock: 
Apparently, up until now, we have been always setting `ai_flags` to `0`.
This was a mistake I introduced when trying to fix some address resolution issues when IPv6 and IPv4 where both active.
This might introduce regressions on some platform, we might just go for `ai_flags = 0` always since it always worked for us, but I figure it would be better to actually let the system recognize the preferred IP type if that does not cause troubles.
:alarm_clock:

Reply:
Since setting it to `0` was a mistake, I'd say let's fix it and see what happens. This should probably not be cherry-picked for *3.0* though to prevent possible regressions in stable - in *master* we can cope with them :)